DxDispatch/src/dxdispatch/HlslDispatchable.cpp

```diff
@@ -333,7 +333,7 @@ void HlslDispatchable::Initialize()
     }
 }
 
-void HlslDispatchable::Bind(const Bindings& bindings, uint32_t iteration)
+void HlslDispatchable::Bind(Bindings& bindings, uint32_t iteration)
```

Comment:
It would be good to throw an exception if the user attempts to bind any deferred resources to an HLSL dispatchable. Same applies to DML dispatchable.

Reply:
ok will update